lib/fromhttp.js

```diff
@@ -80,6 +80,7 @@ function fromHttp(app, queueImpl, opts, ttl) {
 
         if (httpBody) {
             if (parsed.isBinary) httpBody = new Buffer(httpBody, 'base64');
+            continuation.resetTimeout && continuation.resetTimeout();
```

Comment:
Some `continuation`s don't have `resetTimeout`?

Reply:
not really, I think I am being paranoid!

Reply:
Removing this thanks for pointing out.